src/Facade/Services/PartFacadeService.cs

```diff
@@ -64,6 +64,21 @@ public string GetPartNumberFromId(int id)
             }
         }
 
+        public IResult<BomTypeChangeResource> GetBomType(int partNumberId, IEnumerable<string> privileges = null)
```

Comment:
can't you just get this information from the part without having a separate facade method for it?

Reply:
I think it could be a possible refactor unless i remember what pushed me down this route